src/ts/header_generated.ts

```diff
@@ -5,12 +5,21 @@ import { flatbuffers } from 'flatbuffers'
  * @enum {number}
  */
 export enum GeometryType{
-  Point= 0,
-  MultiPoint= 1,
+  Point= 1,
   LineString= 2,
-  MultiLineString= 3,
-  Polygon= 4,
-  MultiPolygon= 5
+  Polygon= 3,
+  MultiPoint= 4,
+  MultiLineString= 5,
+  MultiPolygon= 6,
+  GeometryCollection= 7,
```

Comment:
as GeometryCollection means a multi-geometry of unknown simple types, it could be consistent to also have a 0 = Unknown

Reply:
Agreed, makes sense to have Unknown in several aspects.